.github/workflows/update-flake-lock.yml

```
@@ -35,6 +38,22 @@ jobs:
             echo "changed=true" >> $GITHUB_OUTPUT
           fi
 
+      - name: Build derivation after update
+        if: steps.changes.outputs.changed == 'true'
+        run: nix build .#devShells.x86_64-linux.default --accept-flake-config -o ./after-update
+
+      - name: Install dix
+        if: steps.changes.outputs.changed == 'true'
+        run: nix profile install github:faukah/dix
```

```suggestion
        run: nix profile install nixpkgs#dix
```
can be simplified to this